publish-scripts/chocolatey/installps_template

```diff
@@ -1,23 +1,32 @@
-$ErrorActionPreference = 'Stop';
+$ErrorActionPreference = 'STOP';
 
 $packageName= '$PACKAGENAME'
 $toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$url_arm64      = '$ZIPURL_ARM64'
 $url_x86        = '$ZIPURL_X86'
 $url_x64        = '$ZIPURL_X64'
+$checksum_arm64 = '$CHECKSUM_ARM64'
 $checksum_x86   = '$CHECKSUM_X86'
 $checksum_x64   = '$CHECKSUM_X64'
 
-# By default, we want 32-bit installer
-$url = $url_x86
-$checksum = $checksum_x86
-
-# If specifically asked for 64-bit, we use that
+# If specifically asked for x64 or arm64, we use that
 $pp = Get-PackageParameters
```

Comment:
If `Get-PackageParameters` returns a hashtable, we could use `$pp.ContainsKey($keyName)`.